geopandas/tests/test_overlay.py

```diff
@@ -166,33 +165,32 @@ def test_overlay_nybb(how):
         assert len(result.columns) == len(expected.columns)
         result = result.reindex(columns=expected.columns)
 
-    if compat.USE_PYGEOS and how in ("symmetric_difference", "union"):
-        # the ordering of the spatial index results causes slight deviations
-        # in the resultant geometries for multipolygons
-        # for more details on the discussion, see:
-        # https://github.com/geopandas/geopandas/pull/1338
-        # https://github.com/geopandas/geopandas/issues/1337
-
-        # Temporary workaround below:
-
-        # simplify multipolygon geometry comparison
-        # since the order of the constituent polygons depends on
-        # the ordering of spatial indexing results, we cannot
-        # compare symmetric_difference results directly when the
-        # resultant geometry is a multipolygon
-
-        # first, check that all bounds and areas are approx equal
-        # this is a very rough check for multipolygon equality
-        pd.testing.assert_series_equal(
-            result.geometry.area, expected.geometry.area, check_less_precise=True
-        )
-        pd.testing.assert_frame_equal(
-            result.geometry.bounds, expected.geometry.bounds, check_less_precise=True
-        )
+    # the ordering of the spatial index results causes slight deviations
+    # in the resultant geometries for multipolygons
+    # for more details on the discussion, see:
+    # https://github.com/geopandas/geopandas/pull/1338
+    # https://github.com/geopandas/geopandas/issues/1337
+
+    # Temporary workaround below:
+
+    # simplify multipolygon geometry comparison
+    # since the order of the constituent polygons depends on
+    # the ordering of spatial indexing results, we cannot
+    # compare symmetric_difference results directly when the
+    # resultant geometry is a multipolygon
+
+    # first, check that all bounds and areas are approx equal
+    # this is a very rough check for multipolygon equality
+    pd.testing.assert_series_equal(
+        result.geometry.area, expected.geometry.area, check_less_precise=True
+    )
+    pd.testing.assert_frame_equal(
+        result.geometry.bounds, expected.geometry.bounds, check_less_precise=True
+    )
```

Comment:
We now need to do this for all CI runs. It would be nice to have a reliable way to do these mutlipolygon comparisons.

Reply:
> It would be nice to have a reliable way to do these mutlipolygon comparisons.

Can you open a follow-up issue for that?